### ISSUE TITLE
WMTS optionsFromCapabilities ignores WGS84BoundingBox

### ISSUE THREAD

**Issue by M393** [openlayers/openlayers#11350]
**Describe the bug**
WGS84BoundingBox from WMTSCapabilities.xml is ignored.
There exists a test to check it is read correctly but it is not used anywhere in the source.
Related issues: #10979.

**To Reproduce**
1. Go to https://openlayers.org/en/latest/examples/wmts-hidpi.html
2. Look in network manager when the view is outside the bounding box there are lots of 404 status codes because it tries to load tiles that don't exist.

**Expected behavior**
Only tiles inside the bounding box should be loaded.

**Comment by M393** [comment]
This seems to work. There are now only a few 404's at the edges.
But this shouldn't be necessary, should it?
```js
    const wgs84BoundingBox = result.Contents.Layer.find(function (l) {
      return l.Identifier === layer;
    }).WGS84BoundingBox;
    const extent = boundingExtent([
      fromLonLat(wgs84BoundingBox.slice(0, 2)),
      fromLonLat(wgs84BoundingBox.slice(2, 4)),
    ]);
    map.addLayer(
      new TileLayer({
        source: new WMTS(options),
        extent: extent,
      })
    );
```

**Comment by ahocevar** [comment]
No, that should not be necessary. Would you be able to provide a pull request?

**Comment by M393** [comment]
I don't think I'll be able to provide fix, sorry.

**Comment by FrankyBoy** [comment]
Just tried the code from @M393 and for me the result of parser.read does not even include the three BoundingBox properties any more in the generated object, even though they are clearly there in the XML:

```
<Layer>
    <ows:Identifier>layer_name</ows:Identifier>
    <ows:Title>layer_name</ows:Title>
    <ows:Abstract>layer_name</ows:Abstract>
    <ows:WGS84BoundingBox>
        <LowerCorner>16.135238 48.349748</LowerCorner>
        <UpperCorner>16.255636 48.42425</UpperCorner>
    </ows:WGS84BoundingBox>
    <ows:BoundingBox crs="EPSG:3857">
        <LowerCorner>1796166.507554 6165239.013791</LowerCorner>
        <UpperCorner>1809569.145283 6177727.309223</UpperCorner>
    </ows:BoundingBox>
    <ows:BoundingBox crs="EPSG:31256">
        <LowerCorner>-14593.085712 356831.474681</LowerCorner>
        <UpperCorner>-5660.506093 365132.23973</UpperCorner>
    </ows:BoundingBox>
    <Style isDefault="true">
        <ows:Identifier>default</ows:Identifier>
        <ows:Title>default</ows:Title>
    </Style>
    <Format>image/png</Format>
    <Format>image/jpeg</Format>
    <TileMatrixSetLink>
        <TileMatrixSet>EPSG:3857</TileMatrixSet>
        <TileMatrixSetLimits>
            <TileMatrixLimits>
                <TileMatrix>0</TileMatrix>
                <MinTileCol>0</MinTileCol>
                <MaxTileCol>0</MaxTileCol>
                <MinTileRow>0</MinTileRow>
                <MaxTileRow>0</MaxTileRow>
            </TileMatrixLimits>
            [and so on]
        </TileMatrixSetLimits>
    </TileMatrixSetLink>
    <TileMatrixSetLink>
        <TileMatrixSet>EPSG:31256</TileMatrixSet>
        [again limit definitions as above]
        </TileMatrixSetLimits>
    </TileMatrixSetLink>
</Layer>
```

**Comment by MoonE** [comment]
You are missing the ows namespace for the LowerCorner and UpperCorner nodes:
```xml
    <ows:WGS84BoundingBox>
        <ows:LowerCorner>16.135238 48.349748</ows:LowerCorner>
        <ows:UpperCorner>16.255636 48.42425</ows:UpperCorner>
    </ows:WGS84BoundingBox>
```

**Comment by FrankyBoy** [comment]
Nothing like bugs in third party code -.- Oh well time to talk to the vendor.

**Comment by MoonE** [comment]
Fixed by #11405
